packages/apps/frontera/src/store/Organizations/Organization.store.ts

```diff
@@ -129,6 +131,12 @@ export class OrganizationStore extends Syncable<Organization> {
       );
   }
 
+  get country() {
+    if (!this.value.locations?.[0]?.countryCodeA2) return undefined;
+
+    return countryMap.get(this.value.locations[0].countryCodeA2.toLowerCase());
+  }
+
```

Comment:
I think you can just
```ts
return countryMap.get(this.value.locations[0].countryCodeA2?.toLowerCase());
```
and optional chain .toLowerCase(). It should return undefined in case there's no entry in the map.

Reply:
it requires type assection cause get expects string as parameter. I'm not sure if it's any better than having 1 extra line for undefined check

Reply:
you can map.get(undefined) and it returns undefined. But ok, leave it like that for now.